tests/telemetry/test_telemetry.py

```diff
@@ -253,7 +253,7 @@ def process_trace(self, trace):
 
     events = test_agent_session.get_events()
```

Comment:
We can get rid of the flakiness by just getting the app-stated event. The total number of events will vary so we should avoid asserting it.

```suggestion
    events = test_agent_session.get_events("app-started")
```